packages/admin/admin/src/mui/menu/CollapsibleItem.styles.ts

```diff
@@ -0,0 +1,68 @@
+import { Theme } from "@mui/material";
+import { createStyles } from "@mui/styles";
+
+import { MenuCollapsibleItemProps } from "./CollapsibleItem";
+
+export type MenuCollapsibleItemClassKey =
+    | "root"
+    | "childSelected"
+    | "listItem"
+    | "open"
+    | "collapsedMenuParentTitle"
+    | "itemTitle"
+    | "collapsibleIcon"
+    | "colorGrey"
+    | "colorWhite"
+    | "collapsibleOpen";
```

Comment:
Is `collapsibleOpen` this actually being used?

Reply:
✅